### PR TITLE
fixed setup

### PR DESCRIPTION
**Pull request by fgolemo**
fixed git requirements to have egg tokens - otherwise, they don't install, 
shortened and simplified setup.py, 
black-formatted

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,53 +1,42 @@
...
-            continue
-        if ln:  # if requirement is not empty
-            reqs.append(ln)
-    return reqs
 
 setup(
-    name='sequoia',
-    __version__ = "0.0.1",
+    name="sequoia",
+    version="0.0.2",
```
Comment (lebrice) [inline setup.py:18]: No need to bump the version. This isn't even on pypi yet, but still.

**Review by lebrice — CHANGES_REQUESTED** [review]
Curious, I haven't had any issues installing the dependencies before. My understanding was that since the `setup.py` is also in the root folder of both of those repos, there's no need for specifying the egg manually. Am I missing something? What kind of issues were you running into?
Left only one minor comment about not bumping the version, but apart from that, all good.

File: setup.py (hunk 1 above)
Comment (fgolemo) [reply setup.py:18]: feel free to unbump it. But the version string should be `version`, not `__version__`

**Comment by fgolemo** [comment]
As for the issues installing: yeah it's easy to reproduce ...

- create new empty environment,
- try installing with `pip install -e .` (NOT pip install requirements)
- check if git dependencies are installed -> for me, they are not (e.g. `nngeometry` is missing, but `gym` is installed from Pypi as dependency for something, not your github version)
- then apply patch proposed in this commit, and reinstall
- notice how the github dependencies are now installed correctly.

Let me know if for some reason this doesn't apply to you. Make sure to use a fresh, clean environment (like a virtualenv or a new conda env)

File: setup.py [hunk 2]
```diff
@@ -1,53 +1,42 @@
...
-            "orion",
-            "orion.algo.skopt",
-        ]},
-    install_requires=load_requirements(),
-    python_requires='>=3.7',
-    tests_require=['pytest'],
+        "hpo": ["orion", "orion.algo.skopt",],
+    },
+    install_requires=packages_pip,
+    dependency_links=packages_git,
```
Comment (lebrice) [inline setup.py:33]: This `packages_git` isn't defined anywhere, I just set it to its default (empty list)

**Review by lebrice — APPROVED** [review]
I am unable to reproduce your issue (see below), but since it also still works with this "fix", then I guess we might as well just push it through:

```console
$ conda create -n bob python=3.8
(...)
$ conda activate bob
$ which pip
/home/fabrice/miniconda3/envs/bob/bin/pip
$ pip install -e .
(no errors)
$ which pytest
/home/fabrice/miniconda3/envs/bob/bin/pytest
$ pytest -x -v
(all tests passing after about 5-10 minutes)
```

**Pull request merged**